docs/release/osg-36.md

```diff
@@ -50,7 +50,19 @@ excludepkgs=rrdtool
 Latest News
 -----------
 
-### **March 30, 2023: EL9 and Gratia Probe 2.8.4 (all operating systems)**
+### **April 20, 2023:** HTCondor-CE 6.0.0, htgettoken; Upcoming: HTCondor 10.4.0
+-   [HTCondor-CE 6.0.0](https://htcondor.com/htcondor-ce/v6/releases/#600)
+    -   Align HTCondor-CE security configuration with HTCondor defaults
+    -   Add example configuration on how to ban users
+    -   Add `condor_ce_transform_ads` command
+    -   Improve essential directory checking and creation at startup
+-   htgettoken 1.17
+    -   Make `--showbearerurl` work properly in combination with `--nobearertoken`
+    -   `httokendecode`'s error message for missing token file now goes to `stderr`
+-   Upcoming: [HTCondor 10.4.0](https://htcondor.readthedocs.io/en/latest/version-history/feature-versions-10-x.html#version-10-4-0) - new feature release
```

Comment:
```suggestion
-   EL7/EL8 upcoming and EL9 release: [HTCondor 10.4.0](https://htcondor.readthedocs.io/en/latest/version-history/feature-versions-10-x.html#version-10-4-0) - new feature release
```

Folks mentioned that they were confused by some of the wording in my last announcement so it's good to be explicit here.